src/match.js

```diff
@@ -162,15 +162,17 @@ function findAttributesPattern (priority, element, ignore) {
 
     var pattern = `[${attributeName}="${attributeValue}"]`
 
-    if ((/\b\d/).test(attributeValue) === false) {
```

Comment:
Can we infer (or guess) the reasoning behind testing the attribute against this regex?

Reply:
Optimal Select, unfortunately, has no tests which might give a clue. :( 

The result of this, however, is really weird. If we have a div like:

```html
<div class="column-8 something-else">Hello</div>
```

Then, this condition would fail, generating this selector:

```js
pattern = '[class="column-8 something-else"]';

// Whereas, the expected is:
pattern = '.column-8.something-else';
```

This becomes _especially_ problematic if this is paired with line breaks in class names (which is what Audible does: 

```html
<div class="column-8 


    something-else">Hello</div>
```

That would generate:

```js
pattern = '[class="column-8\
\
\
   something-else"]';
```

Reply:
w. t. f.
o_0